docs/source/checkpoints.md

```diff
@@ -2,7 +2,7 @@
 
 Hi-ml toolbox offers different utilities to parse and download pretrained checkpoints that help you abstract checkpoint
 downloading from different sources. Refer to
-[CheckpointParser](https://github.com/microsoft/hi-ml/blob/main/hi-ml/src/health_ml/utils/checkpoint_utils.py#L238) for
+[CheckpointParser](https://github.com/microsoft/hi-ml/blob/c4e25b8a797cfee3822bda1e1c05e399667bf68e/hi-ml/src/health_ml/utils/checkpoint_utils.py#L238) for
```

Comment:
As mentioned when we spoke, I think it is actually safer to link to main: discrepancies between doc and current code will be more obvious. Consequnetly, the pre-commit settings should be updated.